pilot/pkg/networking/core/v1alpha3/tunnelingconfig/apply.go

```diff
@@ -26,7 +26,7 @@ import (
 type ApplyFunc = func(tcpProxy *tcp.TcpProxy, destinationRule *networking.DestinationRule, subsetName string)
 
 // Apply configures tunneling_config in a given TcpProxy depending on the destination rule and the destination hosts
-var Apply ApplyFunc = func(tcpProxy *tcp.TcpProxy, destinationRule *networking.DestinationRule, subsetName string) {
+var Apply = func(tcpProxy *tcp.TcpProxy, destinationRule *networking.DestinationRule, subsetName string) {
```

Comment:
Seems like this is useful - if the ApplyFunc signature changes this will make it obvious.